hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -253,11 +256,16 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       addResponseToDoubleBuffer(transactionLogIndex, omClientResponse,
           ozoneManagerDoubleBufferHelper);
       if (acquiredBucketLock) {
-        metadataManager.getLock().releaseWriteLock(BUCKET_LOCK, volumeName,
-            bucketName);
+        getOmLockDetails().merge(
```

Comment:
Suggest to have a updateProcessingDetails() function in OMClientRequest to wrapper the getOmLockDetails().merge() logic here.

Reply:
Updated, PTAL